bmorph/evaluation/StatCor_Vis.py

```diff
@@ -0,0 +1,566 @@
+import numpy as np
+import xarray as xr
+import pandas as pd
+import seaborn as sns
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import sys
+import bmorph
+
+sns.set_context('talk')
+mpl.style.use('seaborn-bright')
+mpl.rcParams['figure.figsize'] = (12, 9)
+CFS_TO_CMS = 35.314666212661
+
+import descStats as dst
+import format_data as fd;
+
+colors99p99=['Maroon','Navy','Black','Orange','Yellow','Blue','Grey','Pink','Lavender']
+siteList = fd.sites
+
+def custom_legend(names: list(),colors=colors99p99):
```

Comment:
Your type declaration is `list()` which actually instantiates a new list (sort of, I think the python interpreter won't actually run type annoations) - I think the more idiomatic way to do this is to do:
```
#Up at the top
from typing import List
#All the other code goes here
#....
def custom_legend(names: List, colors=colors99p99):
```